tests/resources/nimbus-testbeds/vic-robo.rb

```diff
@@ -0,0 +1,72 @@
+#usage: nimbus-testbeddeploy --testbedSpecRubyFile ~/vcan-vc-5esx-allflash.rb \
```

Comment:
this is at best a partially working solution with quite a few errors in it looks like... I would rather not merge this into master at this point. Better to pull this out of this PR and merge it later when it is fixed.

Reply:
I guess this PR is not actually to master, but don't really think this file in it's current state needs to be in the feature branch - but I won't block the PR if you really want to merge it into the feature branch for some reason.

Reply:
@mhagen-vmware I actually am planning to use this going forward. The only thing blocking me on that part is putting it on dbc so that the it's accessible on nimbus.  Its definitely not perfect, but its better than using the simple vSAN testbed (faster, no NFS, more control over ESXs & room to grow into an ELM deployment). The only thing it currently does not do that it should is enable HA, but govc can take care of that if we can't ultimately get it done in the testbed spec. 

Can you point out the errors so that I can fix them?